1.10/deploying-services/containerizers/ucr.md

```diff
@@ -24,21 +24,25 @@ The Universal Container Runtime offers the following advantages:
 
 To provision containers with the UCR from the DC/OS CLI, specify the container type `MESOS` and a the appropriate object in your [Marathon application definition](/docs/1.10/deploying-services/creating-services/). Here, we specify a Docker container with the `docker` object.
```

Comment:
Line 8 of the unchanged content, is missing a space between `Universal Container Runtime(UCR)`.